docs/spec.md

```diff
@@ -2593,6 +2661,14 @@ planning to address this in
   * If `!is_quantized(lhs)`:
     * (C20) `element_type(lhs) = expressed_type(rhs) = element_type(result)`.
 
+* (C21) If `precision_config... != DEFAULT`,
```

Comment:
I'm struggling on deciding how to spec these. Because really we want their values to be empty, but we don't have a notion of optional in our spec... I.e. `DEFAULT` doesn't actually mean that `lhs_precision_type = element_type(lhs)` on all hardware.. it's meaning is impl defined though so its hard to give a true default value.

I don't think there's a use case for explicitly specifying the precision types to be exactly these, but still risks some ambiguity?

Reply:
discussed offline, will use `None`.